src/execution/execute-afm.ts

```diff
@@ -113,18 +118,23 @@ export class ExecuteAfmModule {
         });
     }
 
-    private getOnePage(pollingUri: string, offset: number[], prevResult: any = null): any {
-        return this.fetchExecutionResult(pollingUri, offset).then((executionResult: Execution.IExecutionResult) => {
-            if (executionResult === null) {
-                return null;
-            }
-
-            const newResult = prevResult ? mergePageData(prevResult, executionResult) : executionResult;
-
-            const nextOffset = nextPageOffset(executionResult.executionResult.paging);
-            return nextOffset
-                ? this.getOnePage(pollingUri, nextOffset, newResult)
-                : newResult;
-        });
+    private getOnePage(
+        pollingUri: string, offset: number[], prevResultWrapper?: Execution.IExecutionResultWrapper
```

Comment:
there was for prevResultWrapper set a default value (null) its intentional change?

Reply:
`Type 'null' is not assignable to type 'IExecutionResultWrapper'.` :)